run-create/README.md

```diff
@@ -0,0 +1,66 @@
+# Create Test Run
+
+This action creates a new test run in Qase TMS and returns its ID.
+
+## Inputs
+
+### token
+
+**Required**. Qase API token. You can get it in your account settings.
+
+### project
+
+**Required**. Project code where the test run will be created.
+
+### title
+
+**Required**. Title of the test run.
+
+### description
+
+Description of the test run.
+
+### environment
+
+Environment of the test run.
+
+### milestone
+
+Milestone of the test run.
```

Comment:
```suggestion
Milestone of the test run, specified by ID.
```

Reply:
Do you want to specify ID of env, milestone, plan?

Reply:
Fixed